AsyncAwait.Tasks/AsyncAwait.Task1.CancellationTokens/Program.cs

```diff
@@ -31,7 +38,15 @@ private static void Main(string[] args)
         {
             if (int.TryParse(input, out var n))
             {
-                CalculateSum(n);
+                await CalculateSum(n);
+            }
+            else if(input.Trim().ToUpper() == "C")
+            {
+                // todo: add code to process cancellation and uncomment this line    
+                // Console.WriteLine($"Sum for {n} cancelled...");
+                source.Cancel(); 
+                token.ThrowIfCancellationRequested();

```

Comment:
You shouldn't throw exception here, the idea of the exercise is that Calculator waits for token cancellation and handles it, in Program you just need to call Cancel (as you already did on line above)